test/e2e/lib/gutenberg/gutenberg-editor-component.js

```diff
@@ -336,11 +336,21 @@ export default class GutenbergEditorComponent extends AsyncBaseContainer {
 	}
 
 	async launchPreview() {
-		return await driverHelper.clickWhenClickable(
+		if ( driverManager.currentScreenSize() === 'mobile' ) {
```

Comment:
We might have to revert some of these changes if we don't end up using the device switcher preview button in 7.7.

The [media queries simulator magic](https://github.com/WordPress/gutenberg/blob/master/packages/block-editor/src/components/use-simulated-media-query/index.js#L18) won't work X-domain.

See: p7DVsv-8gl-p2#comment-27501

Reply:
I commented out/simplified the desktop version but left this mobile `if` in place since we'll need later anyway.